src/plugins/orangehrmRecruitmentPlugin/Dao/RecruitmentAttachmentDao.php

```diff
@@ -191,8 +215,11 @@ public function getInterviewAttachments(
     protected function getInterviewAttachmentPaginator(
         InterviewAttachmentSearchFilterParams $interviewAttachmentSearchFilterParams
     ): Paginator {
-        $qb = $this->createQueryBuilder(InterviewAttachment::class, 'attachment');
-        $qb->where('attachment.interview = :interviewId')
+        $select = 'NEW ' . RecruitmentAttachment::class . "(interviewAttachment.id,interviewAttachment.fileName,interviewAttachment.fileType,interviewAttachment.fileSize,interview.id,interviewAttachment.comment)";
+        $qb = $this->createQueryBuilder(InterviewAttachment::class, 'interviewAttachment');
+        $qb->leftJoin('interviewAttachment.interview', 'interview');
```

Comment:
```suggestion
```
I forgot to ask you to revert this change.
use `IDENTITY(interviewAttachment.interview)`